docs/user-guide/config-examples.md

```diff
@@ -0,0 +1,30 @@
+# User configurations
+
+Want some help with starting your configuration ?
+Have a look at this catalog of configuration examples.
```

Comment:
Subjective (other than space before `?`), feel free to ignore or iterate:

```suggestion
Want some inspiration for your configuration?
Or maybe you're curious how others have done things?
Take a look at these configuration examples below.
```

Reply:
I agree, this is better !